2.0/0x200-Components/0x208-C8-Liquidity-pool.md

```diff
@@ -0,0 +1,27 @@
+# C8: Liquidity pool
```

Comment:
Add it on the main page to the list (README.md).

Reply:
Added.